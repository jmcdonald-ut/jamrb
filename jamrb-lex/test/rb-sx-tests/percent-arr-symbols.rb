%i[sym1 sym2 sym3 sym4]
