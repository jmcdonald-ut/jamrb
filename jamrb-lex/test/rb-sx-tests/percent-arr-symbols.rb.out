((Token ((1 0) on_qsymbols_beg "%i["))
 (Token ((1 3) on_tstring_content "sym1"))
 (Token ((1 7) on_words_sep " "))
 (Token ((1 8) on_tstring_content "sym2"))
 (Token ((1 12) on_words_sep " "))
 (Token ((1 13) on_tstring_content "sym3"))
 (Token ((1 17) on_words_sep " "))
 (Token ((1 18) on_tstring_content "sym4"))
 (Token ((1 22) on_words_sep "]"))
 (Token ((1 23) on_nl "\n")))
